### PR TITLE
chore: Enable spaced-comment lint rule

### PR DESCRIPTION
**Pull request by patrickhousley**
### Overview

Enabling the `spaced-comment` lint rule and running auto fixes.
### Related Issue(s)

https://issues.newrelic.com/browse/NR-87756

### CODE REVIEW

**Comment by cwli24** [comment]
Lol this one is so asymmetric

**Review by metal-messiah — APPROVED** [review]

File: src/features/page_view_timing/aggregate/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class Aggregate extends AggregateBase {
     this.curSessEndRecorded = false
     this.cls = null // this should be null unless set to a numeric value by web-vitals so that we differentiate if CLS is supported
 
-    /*! This is the section that used to be in the loader portion: !*/
+    /*! This is the section that used to be in the loader portion: ! */
```
Comment (bjfield) [inline src/features/page_view_timing/aggregate/index.js:32]:
```suggestion
    /* ! This is the section that used to be in the loader portion: ! */
```

File: tools/testing-server/test-handle.js [hunk 2]
```diff
@@ -274,7 +274,7 @@ module.exports = class TestHandle {
     return { promise, resolve: capturedResolve, reject: capturedReject }
   }
 
-  /****** BAM Expect Shortcut Methods ******/
+  /** **** BAM Expect Shortcut Methods ******/
```
Comment (bjfield) [inline tools/testing-server/test-handle.js:277]:
```suggestion
  /* ***** BAM Expect Shortcut Methods ***** */
```

File: tools/wdio/plugins/testing-server/test-handle-connector.mjs [hunk 3]
```diff
@@ -98,7 +98,7 @@ export class TestHandleConnector {
     }
   }
 
-  /****** BAM Expect Shortcut Methods ******/
+  /** **** BAM Expect Shortcut Methods ******/
```
Comment (bjfield) [inline tools/wdio/plugins/testing-server/test-handle-connector.mjs:101]:
```suggestion
  /** **** BAM Expect Shortcut Methods ******/
```
```suggestion
  /* ***** BAM Expect Shortcut Methods ***** */
```

**Review by bjfield — APPROVED** [review]

**Pull request merged**